message_ix_models/model/material/build.py

```diff
@@ -169,16 +129,31 @@ def build(
         last_hist_year = scenario.par("historical_activity")["year_act"].max()
         modify_industry_demand(scenario, last_hist_year, iea_data_path)
         add_new_ind_hist_act(scenario, [last_hist_year], iea_data_path)
-        add_elec_i_ini_act(scenario)
         add_emission_accounting(scenario)
 
-        # scenario.commit("no changes")
+    if modify_existing_constraints:
+        calibrate_existing_constraints(scenario)
+
+    return scenario
+
+
+def calibrate_existing_constraints(scenario):
```

Comment:
It seems this whole function is not covered by tests. Does time allow you to add a test that calls this function (maybe with two scenarios, one with a model including `"SSP_dev"`, one without such a model)?

Reply:
Same problem. To my knowledge we do not have code that can create a suitable scenario nor a loadable snapshot at the moment.